src/bun.js/node/path_watcher.zig

```diff
@@ -168,6 +181,8 @@ pub const PathWatcherManager = struct {
 
                         for (watchers) |w| {
                             if (w) |watcher| {
+                                if (watcher.fsevents_watcher != null) continue;
```

Comment:
this is a macOS only thing right? could we add a `if (comptime Environment.isMac)` here?

Reply:
right i forgot that my bad